src/index.js

```diff
@@ -1,26 +1,29 @@
 import Vue from "vue";
 import VueRouter from "vue-router";
+import "animate.css";
 import App from "./App.vue";
 import router from "./helpers/router";
 import { store } from "./helpers/store";
 import * as fb from "./helpers/firebaseConfig";
 import Icon from "vue-awesome/components/Icon.vue";
+import VueMoment from "vue-moment";
 
 Vue.config.productionTip = false;
 Vue.config.devtools = process.env.NODE_ENV === "development";
 
+Vue.use(VueMoment);
 Vue.use(VueRouter);
 Vue.component("v-icon", Icon);
 
 new Vue({
   router,
   store,
   created() {
-    fb.auth.onAuthStateChanged((user) => {
+    fb.auth.onAuthStateChanged(async (user) => {
```

Comment:
Note that `async` arrow functions return a `Promise`, which is definitely a different type than the original `(user) => { ... }` function. Is this intended? Does `fb.auth.onAuthStateChanged` expect a `Promise` / `async` callback?

Reply:
See [here](https://firebase.google.com/docs/reference/js/firebase.auth.Auth#onauthstatechanged), honestly I'm not really sure about this. Is there a better way?